helm-chart/splunk-kubernetes-logging/templates/daemonset.yaml

```diff
@@ -74,10 +74,10 @@ spec:
       volumes:
       - name: varlog
         hostPath:
-          path: /var/log
-      - name: varlibdockercontainers
+          path: {{ .Values.containers.path }}
+      - name: varlogdest
         hostPath:
-          path: {{ with .Values.dockerPath }}{{ printf "%s/containers" . | quote }}{{ end }}
```

Comment:
@chaitanyaphalak, was a little confused what to do with this line here. Happy to use .Values.dockerPath but having it append "/containers" automatically won't work for me, I need this to be set to "/var/data/kubeletlogs"

Reply:
I am ok with your implementation. Its more generic.